papers/oliver_beckstein/oliver_beckstein.rst

```diff
@@ -83,6 +83,83 @@ Introduction
 
+Basic Usage
+-----------
+
+The core object in MDAnalysis is the Universe which acts as a nexus for accessing all data contained within the simulation.
+It is initialised by passing the filenames of the topology and trajectory files,
+with a multitude of different formats are supported in these roles.
+The topology acts as a description of all the particles in the system while the trajectory describes their behaviour over time.
+
+.. show loading a Universe and creating basic selections
+.. check that this selection makes chemical sense!
+.. code-block:: python
+  import MDAnalysis as mda
+
+  # Create a Universe based on simulation results
+  u = mda.Universe('topol.top', 'traj.trr')
+
+  # Create a selection of atoms to work with
+  ag = u.atoms.select_atoms('name Ca and not resname MET LYS')
+
+The select_atoms method allows for AtomGroups to be created using 
+a human readable syntax which
+allows queries according to properties, logical statements and geometric criteria.
+
+.. more selection examples, these include
+.. logic operations (NOT AND)
+.. geometry based (AROUND)
+.. other group based (GROUP)
+.. TODO (maybe): brackets, OR, cylinder/sphere?
+.. code-block:: python 
+  # Select all solvent within a set distance from protein atoms
+  ag = u.select_atoms('resname SOL and around 5.0 protein')
+
+  # Select all heavy atoms in the first 20 residues
+  ag = u.select_atoms('resid 1:20 not prop mass < 10.0')
+
+  # Use a preexisting AtomGroup as part of another selection
+  sel1 = u.select_atoms('name N and not resname MET')
+  sel2 = u.select_atoms('around 2.5 group Nsel', Nsel=sel1)
+
+  # Perform a selection on another AtomGroup
+  sel1 = u.select_atoms('around 5.0 protein')
+  sel2 = sel1.select_atoms('type O')
+
+The AtomGroup acts as a representation of a group of particles,
+with the properties of these particles made available as NumPy arrays.
+
+.. accessing data from an atomgroup
+.. topology data
+.. trajectory data
+.. code-block:: python
+  ag.names
+  ag.charges
+  ag.positions
+  ag.velocities
+  ag.forces
+
+The data from MD simulations comes in the form of a trajectory which is a frame by frame description of the motion of particles in the simulation.
+The trajectory data can often reach sizes of 10s of Gb and so reading this entirely into memory would be both slow and impractical.
+Instead, it is made accessible through the trajectory attribute of the Universe.
```

Comment:
It is unclear here that the trajectory attribute helps at this point.  You haven't properly introduced the trajectory yet.  I would rather first introduce our solution that we only ever load one frame into memory and that the trajectory attribute is our interface to the currently loaded frame, as extra goodie we made it behave like a list so that common python idoms for looping can be used.